cgroup_exporter_test.go

```diff
@@ -78,7 +78,7 @@ func TestMetricsHandler(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Unexpected error GET /metrics: %s", err.Error())
 	}
-	if !strings.Contains(body, "cgroup_memory_used_bytes{cgroup=\"/user.slice/user-20821.slice\"} 2.711552e+07") {
```

Comment:
This change is causing unit tests to fail in CirclCI

Reply:
I see what the issue is. I was running the tests manually on a machine with the legacy hierarchy, and I saw the fixture this text was relying on and updated the value to get the tests to pass. The CircleCI must be using unified. I can just revert the change, or we could modify the value to be the same for either hierarchy.

Reply:
Ya CircleCI ends up running tests on CgroupV2 hardware.  I think making the value match across v1 and v2 is likely best option.